client/src/App.js

```diff
@@ -1,20 +1,49 @@
-import React from "react";
+import React, {Component} from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
+import LogIn from './pages/LogIn'
+import SignUp from './pages/SignUp'
+import Profile from './pages/Profile'
 
 import "./App.css";
 
-function App() {
-  return (
-    <MuiThemeProvider theme={theme}>
-      <BrowserRouter>
-        <Route path="/" component={LandingPage} />
-      </BrowserRouter>
-    </MuiThemeProvider>
-  );
+//this needs state
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
```

Comment:
Why does it need state?

Reply:
The state is just there for down the line, when we need to integrate it with the back-end for front end validation, I figure I'm going to need it.

Reply:
I don't think you will need it in this specific component though. You can leave that out for now